### PR TITLE
update minimum system requirements re regular USB keys #241

### LINKED ISSUE #241: update minimum system requirements re regular USB keys
Thanks to forum member b8two for highlighting our messaging failure re regular (read slow) USB key as system disk. It appears from this feedback that it can be interpreted that our current documentation wording does not make clear that absolute function failure will result when an inappropriate system disk is used (Web-UI and underlying support system time outs), i.e. in the case of a regular USB keys use rather than our current wording of:

> 16GB drive for Rockstor's system disk; if USB key use only fast variants (32GB+ SSD recommended).

This issue's author proposes that we add, akin to the VM note, something along the lines of the following:

> Rockstor cannot work reliable when installed on regular USB keys. Only fast variants, or preferably HDD, SSD class devices, are appropriate.

### PR DESCRIPTION
**Pull request by phillxnet**
Add a USB advisory sub-section, and similarly move our existing VMware 'note' to it's own similar advisory sub-section.

## Includes:
- Normalising on space before si units.
- Add note on raid below btrfs.
- Add note on favouring hardware or at least hardware passthrough.
- Improve cpu specificity.
- Minor rewording of this section.

Fixes #241
@FroggyFlox Ready for review. We can always re-visit thought.

- [x] With the proposed changes no Sphinx errors or warnings are generated.
- [x] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

File: installation/quickstart.rst [hunk 1]
```diff
@@ -13,27 +13,42 @@ install.
...
-recommendations.
-
-* 64-bit Intel/AMD, or ARM64 processor (2+ cores recommended).
-* 2GB RAM (4GB+ recommended).
-* 16GB drive for Rockstor's system disk; if USB key use only fast variants
-  (32GB+ SSD recommended).
-* One or more additional drives for data use only (less than 1GB ignored, 5GB+
-  recommended).
+Rockstor is a complete Linux distribution "Built on openSUSE" intended for direct hardware installation.
+Virtual Machine installs can work but are not recommended without full drive or preferably hole drive controller pass-through.
```
Comment (FroggyFlox) [inline installation/quickstart.rst:17]: Typo:
```suggestion
Virtual Machine installs can work but are not recommended without full drive or preferably whole drive controller pass-through.
```

File: installation/quickstart.rst [hunk 2]
```diff
@@ -13,27 +13,42 @@ install.
...
+Hardware raid underneath btrfs, Rockstor's chosen filesystem, will weaken data integrity assurances.
+Raid controllers, if used, should be configured to HBA / JBOD operation.
```
Comment (FroggyFlox) [inline installation/quickstart.rst:19]: Minor style edit
```suggestion
Raid controllers, if used, should thus be configured to HBA / JBOD operation.
```

File: installation/quickstart.rst [hunk 3]
```diff
@@ -13,27 +13,42 @@ install.
...
+
+* x86_64 celeron/AMD equiv, or ARM64 A53 processor (2+ cores i3+ or A72+ recommended).
```
Comment (FroggyFlox) [inline installation/quickstart.rst:21]: Minor style edit:
```suggestion
* x86_64 celeron/AMD equivalent, or ARM64 A53 processor (2+ cores i3+ or A72+ recommended).
```

File: installation/quickstart.rst [hunk 4]
```diff
@@ -13,27 +13,42 @@ install.
...
+.. _usbwarning:
 
-.. [*] For VMware ensure you have :code:`disk.EnableUUID="true"` in your .vmx
-   file.
+USB advisory
+^^^^^^^^^^^^
+
+.. warning::
+
+    Rockstor cannot work reliable when installed on regular USB keys.
```
Comment (FroggyFlox) [inline installation/quickstart.rst:40]: Typo:
```suggestion
    Rockstor cannot work reliably when installed on regular USB keys.
```

File: installation/quickstart.rst [hunk 5]
```diff
@@ -13,27 +13,42 @@ install.
...
+    If USB is used then 2.0 is bare minimum and 3.0 and better is recommended.
+
+.. _vmwarewarning:
+
+VMware advisory
+^^^^^^^^^^^^^^^
+
+.. warning::
+
+    For VMware ensure you have :code:`disk.EnableUUID="true"` in your .vmx file.
```
Comment (FroggyFlox) [inline installation/quickstart.rst:51]: Is it VMWare or VMware?

**Review by FroggyFlox — CHANGES_REQUESTED** [review]
@phillxnet , Thanks for adding this very important information!

I only have a few minor typos and style edits.

File: installation/quickstart.rst (hunk 1 above)
Comment (phillxnet) [reply installation/quickstart.rst:17]: Oh dear; and thanks.

File: installation/quickstart.rst (hunk 4 above)
Comment (phillxnet) [reply installation/quickstart.rst]: What's wrong with me !!

File: installation/quickstart.rst (hunk 5 above)
Comment (phillxnet) [reply installation/quickstart.rst:51]: The latter so I think on this one only, we are good.

**Review by FroggyFlox — APPROVED** [review]

**Comment by phillxnet** [comment]
@FroggyFlox Thanks for the review. And my apologies for so many errors. But we got there in the end.
VMware references from their website:

https://www.vmware.com/

Note blue text in their header.

**Comment by phillxnet** [comment]
Pr product in production.

**Pull request merged**